Makefile

```diff
@@ -416,6 +411,18 @@ endif
 	mkdir -p $(DESTDIR)$(sysconfdir)
 	cp -R $(build_sysconfdir)/julia $(DESTDIR)$(sysconfdir)/
 
+install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html/en/index.html
```

Comment:
doesn't appear to need `stringreplace`, but would usually be best to list `install-nodoc` here instead of separately (on the next line)

Reply:
> doesn't appear to need stringreplace

True.

> but would usually be best to list install-nodoc here instead of separately (on the next line)

Mainly because I don't want to follow the dependency chain and check if there's any conflict between `install-nodoc` and `$(BUILDROOT)/doc/_build/html/en/index.html`. I can certainly move it if you are confident that there isn't any =).

Reply:
conflict -> race

Reply:
eh, that's usually why it should be in the list in the first place: it lets `make` figure out the possible conflicts.

True though that there is a race. Can you make the `install` target depend on `all` also (that should also improve build times on CI slightly)?